javascript/javascript1/week4/Voice_assistant.js

```diff
@@ -0,0 +1,114 @@
+
+let name = null;
+let toDoList = {};
+let timer = 15;
+
+function getReply(command){
+    const command_1 = 'Hello my name is ';
+          command_2 = 'What is my name';
+          command_3 = 'Add fishing to my todo';
+          command_4 = 'Add singing in the shower to my todo';
+          command_5 = 'Remove fishing from my todo';
+          command_6 = 'What is on my todo?';
+          command_7 = 'What day is it today?';
+          command_8 = 'What is ';
+          command_9 = 'Set a timer for 15 seconds';
+
+    if (command.slice(command_1.length) === name){
+        return `We have one more ${name}`;
+    }else if (command.includes(command_1)){
+        name = command.slice(command_1.length);
+        return `Nice to meet you ${name}`;    
+    }
+
+    if (name === null){
+        return `We don't know you`;
+    }else if(command.includes(command_2)){
+        return `Your name is ${name}`;
+    }
+    
+   if (command.includes(command_3)){
+        toDoList[1] =  'fishing';
+    }
+
+    if (command.includes(command_3)){
+        toDoList[1] =  'fishing';
+        return `${toDoList[1]} added to your todo`;
+    }
+
+    if (command.includes(command_4)){
+        toDoList[2] =  'singing in the shower';
+        return `${toDoList[2]} added to your todo`;
+    }
+
+    if (command.includes(command_4)){
```

Comment:
copy pasted `if` statement. It's example of complexity of the whole thing, so even as author you have a risk of making mistakes